examples/code-splitting-native-import-context/README.md

```diff
@@ -0,0 +1,394 @@
+# example.js
+
+This example illustrates how to leverage the `import()` syntax to create ContextModules which are separated into separate chunks for each module in the `./templates` folder.
+
+``` javascript
+
+async function getTemplate(templateName) {
+	try {
+		let template = await import(`./templates/${templateName}`);
+		console.log(template);
+	} catch(err) {
+		console.error("template error");
+		return new Error(err);
+	}
+}
+
+getTemplate("a");
+getTemplate("b");
+getTemplate("c");
+```
+
+# templates/
+
+* a.js
+* b.js
+* c.js
+
+All templates are of this pattern:
+
+``` javascript
+module.exports = function() {
+	return "This text was generated by template X";
+}
```

Comment:
It'd be better to use an `export default` and log `template.default` instead (or destructure `{ default: template }`).

Not requiring the destructuring might actually be a spec violation, even when importing a commonjs module.